test/extended/apiserver/api_requests.go

```diff
@@ -278,8 +279,49 @@ var _ = g.Describe("[sig-arch][Late]", func() {
 			},
 		}
 
-		if _, exists := upperBounds[infra.Spec.PlatformSpec.Type]; !exists {
-			e2eskipper.Skipf("Unsupported platform type: %v", infra.Spec.PlatformSpec.Type)
+		upperBoundsSingleNode := map[configv1.PlatformType]platformUpperBound{
+			configv1.AWSPlatformType: {
+				"authentication-operator":                308,
+				"aws-ebs-csi-driver-operator":            142,
+				"cloud-credential-operator":              64,
+				"cluster-autoscaler-operator":            44,
+				"cluster-baremetal-operator":             31,
+				"cluster-image-registry-operator":        119,
+				"cluster-monitoring-operator":            35,
+				"cluster-node-tuning-operator":           39,
+				"cluster-samples-operator":               23,
+				"cluster-storage-operator":               202,
+				"console-operator":                       146,
+				"csi-snapshot-controller-operator":       99,
+				"dns-operator":                           59,
+				"etcd-operator":                          164,
+				"ingress-operator":                       371,
+				"kube-apiserver-operator":                260,
+				"kube-controller-manager-operator":       145,
+				"kube-storage-version-migrator-operator": 68,
+				"machine-api-operator":                   48,
+				"marketplace-operator":                   12,
+				"openshift-apiserver-operator":           257,
+				"openshift-config-operator":              50,
+				"openshift-controller-manager-operator":  180,
+				"openshift-kube-scheduler-operator":      179,
+				"prometheus-operator":                    90,
+				"service-ca-operator":                    131,
+			},
+		}
+
+		var upperBound platformUpperBound
+
+		if infra.Status.ControlPlaneTopology == v1.SingleReplicaTopologyMode {
```

Comment:
Maybe a comment / commit message explaining why SNO needs higher bounds? I guess even "we're not sure" will be better than nothing. Since it's not really obvious.

Reply:
All the upper bounds are measured experimentally. So "we are not sure" in any cases. It's all a guess based on the CI measurements. The entire test should be treated as such. The main idea is to have a test which will tell us if the upper bound was exceeded ad more importantly how much. To see if the increase is linear or exponential.

Reply:
Fair enough